frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -56,7 +60,7 @@ export function createResources(resources) {
 export function deleteResource(resource) {
   const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
   execSync(
-    `kubectl delete -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
+    `kubectl delete --ignore-not-found=true -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
```

Comment:
the same. Where is this used?

we could also pass an additional argument for this if needed

Reply:
If the resource is not created successfully, the test is failed earlier and will have an error in tear down step in `deleteResource`. This is trying to make it not throwing error, as it seems always been used in tear down step.

Reply:
ok, can we at least add the additional optional argument ignoreNotFound ?

Reply:
Added additional optional argument `ignoreNotFound` with default value `true` because it's used in tear down, in case it's called somewhere else, it can specify it to `false`.

Reply:
+1